README.md

```diff
@@ -1 +1,54 @@
-# bookstore-react
\ No newline at end of file
+# Book Store
+
+> The website allows users to keep record of books and update book data.
+
+## Built With
+
+- Major languages: React
+- linters
+- Frameworks: React
+
+## Live Demo (if available)
+
+[Webpage Link]()
+<br/>
+[Webpage Link]()
+
```

Comment:
- [ ] Please, consider fixing these duplicated links. They are the same and broken. You can delete them for this milestone, while you will deploy your project in the future.